plugin/Faq.pm

```diff
@@ -0,0 +1,34 @@
+package Polebot::Plugin::Faq;
+use strict;
+use warnings;
+use Config::Tiny;
+
+use base 'Polebot::Plugin::Base';
+
+my $cfg = Config::Tiny->read('config');
+my $filename = $cfg -> {"Plugin::Faq"} -> {filename}; 
+open my $fh, "<", $filename or die "Non posso aprire il $filename: $!"; 
+
+my @faq = <$fh>;
+close $fh;
+ 
+sub description { 'FAQ Permette di memorizzare parole con relativa descrizione' }
+ 
+sub public {
```

Comment:
L'indentazione è inconsistente. Io utilizzo il seguente file di configurazione per perltidy:

    -se         # Errors to standard error output
    -w          # Show all warnings
    -i=3        # Three columns indentation
    -ole=unix   # Unix newlines, please
    -pt=2       # Tight paretheses
    -sbt=2      # and square brackets
    -bt=2       # and curly as well
    -nsfs       # My usual style for `for' loops
    -csc        # Closing side comments seem good
    -l=75       # Stand clear from the edge

Ad esempio, riga 25 indenti di un carattere anziché 2 come fai in 19, 20, 21 ma poi indenti di 11 e di 9...